plugins/couchbase_logs.yaml

```diff
@@ -0,0 +1,253 @@
+version: 0.0.1
+title: Couchbase
+description: Log parser for Couchbase
+parameters:
+  - name: enable_error_log
+    type: bool
+    default: true
+  - name: error_log_path
+    type: "[]string"
+    default:
+      - "/opt/couchbase/var/lib/couchbase/logs/error.log"
+  - name: enable_debug_log
+    type: bool
+    default: false
+  - name: debug_log_path
+    type: "[]string"
+    default: 
+      - "/opt/couchbase/var/lib/couchbase/logs/debug.log"
+  - name: enable_info_log
+    type: bool
+    default: false
+  - name: info_log_path
+    type: "[]string"
+    default: 
+      - "/opt/couchbase/var/lib/couchbase/logs/info.log"
+  - name: enable_access_log
+    type: bool
+    default: true
+  - name: http_access_log_path
+    type: "[]string"
+    default: 
+      - "/opt/couchbase/var/lib/couchbase/logs/http_access.log"
+  - name: enable_internal_access_log
+    type: bool
+    default: false
+  - name: http_internal_access_log_path
+    type: "[]string"
+    default: 
+      - "/opt/couchbase/var/lib/couchbase/logs/http_access_internal.log"
+  - name: enable_babysitter_log
+    type: bool
+    default: true
+  - name: babysitter_log_path
+    type: "[]string"
+    default: 
+      - "/opt/couchbase/var/lib/couchbase/logs/babysitter.log"
+  - name: enable_xdcr_log
+    type: bool
+    default: false
+  - name: xdcr_log_path
+    type: "[]string"
+    default: 
+      - "/opt/couchbase/var/lib/couchbase/logs/goxdcr.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    {{ if .enable_error_log}}
+    filelog/error_logs:
+      include: 
+        {{ range $fp := .error_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline: 
+        line_start_pattern: '^\[[^:]*:[^,]*,\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d+-\d{2}:\d{2},[^@]*@[^:]*:'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'couchbase.error'
+        plugin_id: 'Couchbase'
```

Comment:
We've been removing these `plugin_id` fields, we should do the same here to be consistent.